src/Symfony/Component/HttpFoundation/Tests/IpUtilsTest.php

```diff
@@ -19,6 +19,21 @@ class IpUtilsTest extends TestCase
 {
     use ExpectDeprecationTrait;
 
+    public function testWrongProtocolOrder()
```

Comment:
I wrong rather name the test `testSeparateCachesPerProtocol`, to describe what we actually intend to test

Reply:
Yeah, I was struggeling to come up with a name for that test. I will ammend it to use your, clearly much better, name!